src/server.cpp

```diff
@@ -1119,16 +1119,15 @@ PlayerSAO* Server::StageTwoClientInit(u16 peer_id)
 	// Send inventory
 	SendInventory(playersao);
 
-	// Send HP
-	SendPlayerHPOrDie(playersao);
+	// Send HP or death screen
+	if (playersao->isDead())
```

Comment:
They should be functionally the same, but getting rid of getDeadStatus would require storing m_dead in UnitSAO and that means further complications: UnitSAO has the same problem as PlayerSAO in that you can't determine whether or not the unit just died (I think), you just know that the unit is dead. Maybe I should replace `isDead` with `getDeadStatus`? Here it's going to do the same thing, but `getDeadStatus` could potentially get removed later.